ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { Telemetry } from 'types/config.proto';
+
+import { selectors } from 'reducers';
+
+const useAnalytics = () => {
+    const telemetry = useSelector(selectors.publicConfigTelemetrySelector);
+    const { enabled: telemetryEnabled } = telemetry || ({} as Telemetry);
+
+    const analyticsIdentity = useCallback(
+        (userId: string, traits = {}): void => {
+            if (telemetryEnabled) {
+                window.analytics?.identify(userId, traits);
+            }
+        },
+        [telemetryEnabled]
+    );
```

Comment:
Is this recommended by Segment or Amplitude, and will we be using it in the near future?

Reply:
The identity is being set up after initializing segment now. My thought was to keep this around as a basic util function but with it not being used I guess I might as well remove it